src/njoy21/input/GAMINR/Card7/Matd.hpp

```diff
@@ -0,0 +1,15 @@
+struct Matd : public argument::common::Matd {
+  using Value_t = argument::common::Matd::Value_t;
```

Comment:
You don't have to specify this or `name()` because they are defined in the parent class.